src/main/java/com/gojek/beast/worker/Worker.java

```diff
@@ -24,6 +26,11 @@ public void run() {
         do {
             status = job();
         } while (!state.isStopped() && status.isSuccess());
+
+        if (!status.isSuccess()) {
+            statsClient.increment("global.errors,exception=" + status.getException().getClass().getName());
```

Comment:
status has an optional exception. Is it safe to do `getClass()` directly without checking if the exception is set?

Reply:
@kushsharma  you're right. Although incase of FailureStatus the exception would be set, it'd be better to have this check. I've added it.